pkg/pillar/types/zedroutertypes.go

```diff
@@ -1924,12 +1956,12 @@ type NetworkInstanceInfo struct {
 	// interface names for the Logicallabel
 	IfNameList []string // Recorded at time of activate
 
-	// Collection of address assignments; from MAC address to IP address
-	IPAssignments map[string]net.IP
-
 	// Union of all ipsets fed to dnsmasq for the linux bridge
 	BridgeIPSets []string
 
+	// Collection of address assignments; from MAC address to IP address
+	IPAssignments map[string]net.IP
+
```

Comment:
Can you move this back to its original location above BridgeIPSets? No need for uneceesary diffs.